Spanner/src/Transaction.php

```diff
@@ -312,7 +312,9 @@ public function delete($table, KeySet $keySet)
      *
      * Data Manipulation Language (DML) allows you to execute statements which
      * modify the state of the database (i.e. inserting, updating or deleting
-     * rows).
+     * rows). For example, DML supports INSERT, UPDATE and DELETE statements. For
```

Comment:
```suggestion
     * rows). DML supports INSERT, UPDATE and DELETE statements. For
```

Suggest manually applying changes in this paragraph to maintain correct line lengths.